index.js

```diff
@@ -0,0 +1,30 @@
+import Collection from './module/render.js';
+import navAddDisplay from './module/navigation-add-display.js';
+import navContactDisplay from './module/navigation-contact-display.js';
+import navListDisplay from './module/navigation-list-display.js';
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+
+const collection = new Collection();
+
+collection.getBooks();
+
+// event listener to trigger add class
+
+document.querySelector('.addBtn').addEventListener('click', () => {
+  collection.getBooks();
+  collection.addBooks();
+});
+
+// luxon time
+
+const time = DateTime.now();
```

Comment:
- [ ] **Required:** Let's try to implement the Luxon library as shown in the sneak peek. This is designed to help you practice how to work with libraries/frameworks by the following documentation. Take some time to study and play around with [this section of the guide](https://moment.github.io/luxon/#/formatting?id=intldatetimeformat).